modules/getting-started/p_deploy-code.adoc

```diff
@@ -0,0 +1,53 @@
+// Metadata created by nebel
+//
+// UserStory: As an RH SSO customer, I want to deploy specific to complete the initial configuration
+
+[id="deploy-code"]
+= Deploying sample application code
+
+ifeval::[{project_community}==true]
+To make this application functional as a secure application, you install some sample code. You obtain the code by cloning the {quickstartRepo_name} repository at {quickstartRepo_link}. The quickstarts are designed to work with the most recent Keycloak release.
```

Comment:
"you install some sample code" --> "you will need to install the sample code provided".